packages/app/src/cli/api/graphql/business-platform-organizations/queries/fetch_dev_store_by_domain.graphql

```diff
@@ -5,10 +5,10 @@
       properties(filters: {field: STORE_TYPE, operator: EQUALS, value: "app_development"}, search: $domain, offeringHandles: ["shop"]) {
         edges {
           node {
-            __typename
-            id
-            externalId
             ... on Shop {
+              __typename
+              id
+              externalId
```

Comment:
Moved `--typename`, `id` and `externalId` down to the `Shop` level because we only care about `Shop` typed nodes